federation/pkg/federation-controller/util/cluster_util.go

```diff
@@ -146,3 +147,23 @@ func GetClientsetForCluster(cluster *federation_v1beta1.Cluster) (*fedclientset.
 	}
 	return nil, err
 }
+
+// Returns a bool to indicate if the object should be forwarded to a cluster
+func SendToCluster(cluster *federation_v1beta1.Cluster, objMeta metav1.ObjectMeta) bool {
```

Comment:
Consider adding a test targeted at this function to ensure good coverage.  Testing only the controller manager doesn't seem like the right level of abstraction to me.